### ISSUE TITLE
Master Issue for model bringup for tt-mlir feature verification

### ISSUE THREAD

**Issue by tapspatel** [tenstorrent/tt-xla#150]
JAX
1. Write a model in jax
2. Compile using jax.jit

- Showcase vhlo IR (mlir)
- Showcase shlo IR (mlir)
- Showcase TT IR (mlir)
- Showcase TTNN IR (mlir)

3. In visualizer, have different run options

- From vhlo IR
- From shlo IR
- From TT IR
- From TTNN IR

4. In visualizer, showcase golden accuracy information

- For vhlo IR
- For stablehlo IR
- For TT IR
- For TTNN IR

5. In visualizer, showcase performance information

- For vhlo IR
- For stablehlo IR
- For TT IR
- For TTNN IR

6. In visualizer, showcase memory information

- For vhlo IR
- For stablehlo IR
- For TT IR
- For TTNN IR

Sub-Issues to finish goals

- [x] https://github.com/tenstorrent/tt-xla/issues/152
